docs/source/azure/usecases/securityrules.rst

```diff
@@ -0,0 +1,27 @@
+Find Open SSH Ports
+===================
+
+This policy will deny access to all security rules with Inbound SSH ports in the range [8080,8090]
+
+.. code-block:: yaml
+
+     policies:
+       - name: open-ingress
+         resource: azure.networksecuritygroup
+         filters:
+          - type: ingress
+            FromPort: 8080
+            ToPort: 8090
+         actions:
+          - type: close
+
+Available types:
```

Comment:
This documentation type stuff should go into a (not yet existing) \policy\resources\ documentation page dedicated to NSG like they do for custom actions/filters on resources in AWS docs.  We can commit it this way and start on the docs work in a separate commit if you want to avoid adding a bunch more to this one.